### PR TITLE
extract ca code from init.pp

### PR DESCRIPTION
**Pull request by timogoebel**

### CODE REVIEW

**Comment by ekohl** [comment]
There's a merge conflict now.

**Comment by timogoebel** [comment]
@ekohl : Rebased.

File: manifests/ca.pp [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+  $org_unit                = $::certs::org_unit,
+  $ca_expiration           = $::certs::ca_expiration,
+  $generate                = $::certs::generate,
+  $deploy                  = $::certs::deploy,
+  $server_cert             = $::certs::server_cert,
+  $ssl_build_dir           = $::certs::ssl_build_dir,
+  $group                   = $::certs::group,
+  $katello_server_ca_cert  = $::certs::katello_server_ca_cert,
+  $katello_default_ca_cert = $::certs::katello_default_ca_cert,
+  $nss_db_dir              = $::certs::nss_db_dir,
```
Comment (ekohl) [inline manifests/ca.pp]: unused

File: manifests/ca.pp [hunk 2]
```diff
@@ -0,0 +1,118 @@
+# == Class: certs
+# Sets up the CA for Katello
+class certs::ca (
+  $pki_dir                 = $::certs::pki_dir,
```
Comment (ekohl) [inline manifests/ca.pp]: unused

File: manifests/ca.pp [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+  $org                     = $::certs::org,
+  $org_unit                = $::certs::org_unit,
+  $ca_expiration           = $::certs::ca_expiration,
+  $generate                = $::certs::generate,
+  $deploy                  = $::certs::deploy,
+  $server_cert             = $::certs::server_cert,
+  $ssl_build_dir           = $::certs::ssl_build_dir,
+  $group                   = $::certs::group,
+  $katello_server_ca_cert  = $::certs::katello_server_ca_cert,
+  $katello_default_ca_cert = $::certs::katello_default_ca_cert,
```
Comment (ekohl) [inline manifests/ca.pp]: unused

**Comment by timogoebel** [comment]
@ekohl : Updated. Thanks. :-)

**Comment by ehelms** [comment]
Thanks @timogoebel !

**Pull request merged**